README.md

```diff
@@ -688,6 +688,22 @@ client.without_caching do
 end
 ```
 
```

Comment:
`Client#with_caching`

Reply:
Sorry, maybe it's the Github UI, but I'm not sure what you're asking for here. I tried to follow the same kind of formatting used in the README and I dont think I changed anything in this part. Can you clarify?

Reply:
I made an update based on your other comments, but I cant figure this one out - I think this section is from the current README.md